controller/cmd/destination/main.go

```diff
@@ -56,7 +58,11 @@ func Main(args []string) {
 	go func() {
 		log.Infof("starting admin server on %s", *metricsAddr)
 		if err := adminServer.ListenAndServe(); err != nil {
-			log.Errorf("failed to start destination admin server: %s", err)
+			if errors.Is(err, http.ErrServerClosed) {
+				log.Infof("destination admin server status is shutdown or close: %s", err)
```

Comment:
Hm so if we're handling `SIGTERM` we'd end up here, right? I wonder if it's just better to say something like "received shutdown signal" and leave it at that. I'll have to run it myself to see exactly what `err` evaluates to here :D